lib/screens/on_boarding/on_boarding_controller.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:seek_me/general_exports.dart';
+
+class OnBoardingController extends GetxController {
+  int currentIndex = 0;
+  PageController controller = PageController(
+    initialPage: 0,
```

Comment:
I think the default value of `initialPage` is already zero (correct me if I'm wrong) 
If so then we need to remove it